analysis/topEFT/topeft.py

```diff
@@ -530,22 +584,22 @@ def process(self, events):
                 #values=np.asarray(values)
                 hout[var].fill(eft_coeff=eft_coeffs_cut, eft_err_coeff=eft_w2_coeffs_cut, j0eta=values, sample=dataset, channel=ch, cut=lev, sumcharge=sumcharge, weight=weights_flat, systematic=syst)
               elif var == 'e0pt'  : 
-                if ch in ['mmSSonZ', 'mmSSoffZ', 'mmmSSoffZ', 'mmmSSonZ', '4l']: continue
+                if ch in ['mmSSonZ', 'mmSSoffZ', 'mmmSSoffZ', 'mmmSSonZ','mmmm']: continue
```

Comment:
As stated above, the comment above about `2lss` and `4l` categories isn't necessary with any of these, since it is clear these are electron only (or muon for the following lines)